README.md

```diff
@@ -1,2 +1,20 @@
-# hw01
-Repository for first CFSS homework assignment
+# Cheng Yee Lim
+
+![](images/profile_pic.jpg)
+
+> *“A ship is safe in harbor, but that's not what ships are for.”*
+> ― **William G.T. Shedd**
+
+* Singaporean with a severe case of wanderlust
+  + Lived in the United Kingdom for the past three years 
+  + First year in the United States of America
+* Economist by training, but interested to pick up data analytics  
+* To find out more about the coding I do, click [here](www.github.com/limchengyee)
+
+## Process of Editing README.md 
+I first forked my own homework repository from the UC-CFSS. Then, I cloned my repository as a new project in RStudio. Thereafter, I began editing the README.md to include information about myself. Then, I modified the formatting, by including headers, blockquotes, bolding, and italicizing. Whenever I added a new component in, such as a picture, or the blockquote, I made a new commit and pushed it onto GitHub. This allowed me to track how my code developed and allows me to backtrack easily on GitHub. Lastly, I submitted a pull request to merge the changes on my repository (i.e. the modified README.md and the images folder) with the UC-CFSS homework repository. 
```

Comment:
Steps of editing and uploading the markdown document was clear and concise. With regards to the statement: 
"Whenever I added a new component in, such as a picture, or the blockquote, I made a new commit and pushed it onto GitHub"

This can be avoided by clicking the preview button in RStudio so there is no need to constantly commit and push in order to view the component or changes made.